GraphDefinition/GraphDefinition-Genomics-Test-Result.json

```diff
@@ -0,0 +1,181 @@
+{
+  "resourceType": "GraphDefinition",
+  "id": "GraphDefinition-Genomics-Test-Result",
+  "url": "https://fhir.nhs.uk/GraphDefinition/genomics-test-result",
+  "name": "GenomicsTestResult",
+  "status": "draft",
+  "date": "2024-08-09",
+  "publisher" : "NHS England",
+  "description": "GraphDefinition for defining and retrieving the full set of resources relevant to a Genomic Test Result (based on the Genomics Reporting Genomic-Report profile)",
+  "start": "DiagnosticReport",
+  "link": [
+    {
+      "path": "DiagnosticReport.extension.valueReference",
```

Comment:
Not sure if it needs to exist on UKCore or NHSD DiagnosticReport profile but I can't find the extension name, "sliceName": "genomic-study" on either.

Reply:
These are on the genomic reporting IG GenomicReport profile, this is pending further review before officially bringing it into the Genomics IG but this GraphDefinition is just a demonstration of how the extensions could be traversed (the slices should be ignored if they are not found on the DiagnosticReport)